### DISCUSSION TITLE
fuzz test error

### DISCUSSION THREAD

**Discussion by mibiot** [Cyfrin/foundry-full-course-cu#1156] [Q&A]
Can someone help, trying to create fuzz test contract but getting this mistake:

<img width="959" alt="Screenshot 2023-11-28 at 19 02 30" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/140085328/01fec573-0166-4f31-9c65-8c973eb7ca17">

do not understand what's wrong.

**Comment by 0xusmanf** [comment]
Can you please copy paste your code and error instead of a screenshot?
